services/permits/app/celery/elasticsearch_backend.py

```diff
@@ -27,7 +27,7 @@ def _get_server(self):
             http_auth = (self.username, self.password)
 
         return elasticsearch.Elasticsearch(
-            f'{self.scheme}://{self.host}:{self.port}',
+            hosts="https://elasticsearch:9200",
```

Comment:
Is there a way to use dynamic variables here while keeping things working properly?  I know this was part of the change to get it working, but it would be better if we didn't rely on hardcoded values here.

Reply:
Even just using ELASTICSEARCH_HOST

Reply:
oh oops I thought I fixed that!